include/items.php

```diff
@@ -141,11 +141,12 @@ function query_page_info($url, $photo = "", $keywords = false, $keyword_blacklis
 	return $data;
 }
 
-function add_page_keywords($url, $photo = "", $keywords = false, $keyword_blacklist = "")
+function add_page_keywords($url, $photo = "", $keywords = false, $keyword_blacklist = "", $return_array = false)
```

Comment:
Please don't make a function return type depend on a parameter. Either create another function dedicated to this other return format or return a structure with both outputs.

Reply:
This will only be needed during the conversion process. Later on (when we don't use the `term` table anymore), we only return the other value.

Reply:
I get it, but I don't like it one bit.

Reply:
I could copy&paste the whole function - and remove the old function before the release.

Reply:
I'd actually prefer it, this modification looks stealthy enough to stay long past its usefulness.

Reply:
The function is called exactly one time. So I guess it shouldn't happen. But I can try to make you happy :-)